fmriprep/cli/run.py

```diff
@@ -181,6 +179,14 @@ def create_workflow(opts):
         subject_list = [sub[4:] if sub.startswith('sub-') else sub for sub in subject_list]
 
     logger.info('Subject list: %s', ', '.join(subject_list))
+    
+    report_errors = 0
+    if opts.reports_only:
+        for subject_label in subject_list:
+            report_errors += run_reports(settings['reportlets_dir'],
+                                         settings['output_dir'],
+                                         subject_label, run_uuid=run_uuid)
```

Comment:
Since we're not using `report_errors` before exiting in this case want to just make this:

```
if opts.reports_only:
    for subject_label in subject_list:
        run_reports(...)
    sys.exit()
```

And move `report_errors = 0` back to its previous location?

Note that I'm moving `sys.exit()` back one indentation level. As it is, I think you'll only regenerate the report for the first subject.